test/param/loop/paramForLoopWrongTypeHigh.good

```diff
@@ -0,0 +1 @@
+paramForLoopWrongTypeHigh.chpl:1: error: Invalid use of count range in param for loop
```

Comment:
Same comment here as [the one below](https://github.com/chapel-lang/chapel/pull/15552/commits/ea4e72fd399c5437252168f51af7007bd55da31f#r412321002)